### PR TITLE
Add travel fund 2023 transparency report

### LINKED ISSUE #1239: Decide On How Often A Person May Receive Travel Fund Assistance
The initial travel fund work proposed that a person may only receive trave funding once per year. This issue tracks discussing this limit and deciding what, if any limit the CPC wants to impose. 

_Originally posted by @mhdawson in https://github.com/openjs-foundation/cross-project-council/pull/1230#discussion_r1457905210_

### PR DESCRIPTION
**Pull request by tobie**
Closes #1239.

### CODE REVIEW

File: community-fund/COMMUNITY_FUND_POLICY.md [hunk 1]
```diff
@@ -13,6 +13,15 @@ Applicants should explore employer-sponsored funding options as their first sour
...
 Funding decisions are determined by a combination of need and potential impact.
 
+## Transparency
+
+In **2023**, 26 funding requests were made by 22 contributors for a total requested sum of USD 47,500 with a maximum of 2 requests per contributor.
+Requested sums per trip ranged from $750 to $5,750, with a median of $1,525 and an average of $1,825.
+Requested sums per individuals ranged from $1,000 to $7,350, with a median of $1,950 and an average of $2,150.
+All requests were granted.
+
+In 2024, USD 60,000 was allocated to the fund.
```
Comment (ljharb) [inline community-fund/COMMUNITY_FUND_POLICY.md:23]:
```suggestion
In 2024, USD $60,000 was allocated to the fund.
```

**Review by ljharb — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by mcollina** [comment]
Something we should add in the guide for the Travel Fund is to send the final invoice as quickly as possible. This will allow others to use the funds that were allocated but not spent.

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by eemeli — APPROVED** [review]

File: community-fund/COMMUNITY_FUND_POLICY.md [hunk 2]
```diff
@@ -13,6 +13,15 @@ Applicants should explore employer-sponsored funding options as their first sour
 Additionally, 25% of the community fund is allocated specifically to support underrepresented groups.
 Funding decisions are determined by a combination of need and potential impact.
 
+## Transparency
+
+In **2023**, 26 funding requests were made by 22 contributors for a total requested sum of USD 47,500 with a maximum of 2 requests per contributor.
+Requested sums per trip ranged from $750 to $5,750, with a median of $1,525 and an average of $1,825.
+Requested sums per individuals ranged from $1,000 to $7,350, with a median of $1,950 and an average of $2,150.
```
Comment (VoltrexKeyva) [inline community-fund/COMMUNITY_FUND_POLICY.md:20]:
```suggestion
Requested sums per individual ranged from $1,000 to $7,350, with a median of $1,950 and an average of $2,150.
```

File: community-fund/COMMUNITY_FUND_POLICY.md (hunk 1 above)
Comment (tobie) [reply community-fund/COMMUNITY_FUND_POLICY.md:23]: There's a USD prefix on the first one so it's clear these are USD and not dollars from elsewhere.

Comment (ljharb) [reply community-fund/COMMUNITY_FUND_POLICY.md:23]: There’s 4 dollar amounts in line 20 that just use $.

Comment (ljharb) [reply community-fund/COMMUNITY_FUND_POLICY.md]: it looks weird to me to ever see a USD amount without a dollar sign, to be clear :-)

**Review by ljharb — APPROVED** [review]

**Pull request merged**